BREAKING.md

```diff
@@ -169,6 +170,10 @@ have been removed and have no replacement:
 Additionally, the `ISummarizer` interface no longer extends `IFluidLoadable` nor `Partial<IProvideSummarizer>`.
 This means it no longer has readonly properties `IFluidLoadable` and `handle`.
 
+### `MessageType.RemoteHelp` and the foreman lambda was removed
```

Comment:
@Josmithr @vladsud @alexvy86  is this documentation correct? I'm not sure if I should phrase it as it was deprecated and is now being removed because technically there wasn't a merge that went in earlier announcing deprecation with intention to remove. But, functionally it's been deprecated for a while. Or should I split these changes out into two PRs one with intention and the next actually doing the removal?

Reply:
Moved to breaking changes section and updated text to reflect "removal now" instead of "prior depracation and current removal".